scripts/rollup/bundles.js

```diff
@@ -249,7 +249,7 @@ const bundles = [
   /******* React DOM - www - Testing *******/
   {
     moduleType: RENDERER,
-    bundleTypes: [FB_WWW_DEV, FB_WWW_PROD],
+    bundleTypes: [FB_WWW_DEV, FB_WWW_PROD, NODE_DEV, NODE_PROD],
```

Comment:
This adds the `react-dom/testing` entry point to the NPM build. I wish we could only do this for the `__EXPERIMENTAL__` release channel but I don't think that's possible with our current Rollup configuration?

Reply:
Let's rename to `unstable_testing` in this case? We shouldn't ship it in a way that seems usable.

Reply:
Good suggestion.